src/main.py

```diff
@@ -29,10 +29,12 @@ def load_config(self):
         self.config = Config(app.file_path('config.ini'))
 
     def init_piano(self):
-        self.midi = midi.Midi()
+        midi_output_factory = MidiOutputFactory()
         midi_output_driver = self.config.get_midi_output_driver(constants.MIDI_OUTPUT_DEFAULT_DRIVER)
-        if midi_output_driver == constants.MIDI_OUTPUT_DEFAULT_DRIVER:
```

Comment:
O que tu acha de colocar esse if numa função que retorna o driver? Poderia facilitar na hora que a gente for implementar backends por canal